Source/WebKit/UIProcess/mac/WebPageProxyMac.mm

```diff
@@ -919,6 +920,28 @@ static inline bool expectsLegacyImplicitRubberBandControl()
 
 #if ENABLE(WRITING_TOOLS)
 
+bool WebPageProxy::shouldEnableWritingToolsRequestedTool(WebCore::WritingTools::RequestedTool tool) const
+{
+    WTRequestedTool requestedTool = convertToPlatformRequestedTool(tool);
+
+    if (requestedTool == WTRequestedToolIndex)
+        return true;
+
+    bool editorStateIsContentEditable = false;
+    bool allowProofreadingAndRewritingTools = true;
+
+    auto& editorState = this->editorState();
+    editorStateIsContentEditable = editorState.isContentEditable;
```

Comment:
the variable can be declared and defined on this line

Reply:
Done.